extensions/hibernate-validator/deployment/src/main/java/io/quarkus/hibernate/validator/deployment/HibernateValidatorProcessor.java

```diff
@@ -55,7 +82,9 @@ LogCleanupFilterBuildItem logCleanup() {
     }
 
     @BuildStep
-    void registerAdditionalBeans(BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
+    void registerAdditionalBeans(BuildProducer<AdditionalBeanBuildItem> additionalBeans,
+            BuildProducer<AnnotationsTransformerBuildItem> annotationsTransformers,
```

Comment:
BTW, this arg isn't used in this method.

Reply:
Ah, yes, good catch, I removed the annotation transformers I initially added. Will clean it up.

Reply:
@aloubyansky fixed and rebased.